CraftTweaker2-MC1120-Mod-JEI/src/main/resources/META-INF/crafttweaker_at.cfg

```diff
@@ -9,4 +9,7 @@ public-f net.minecraft.client.entity.EntityPlayerSP field_192036_cb #recipeBook
 public net.minecraft.block.Block field_149782_v #blockHardness
 public net.minecraft.creativetab.CreativeTabs field_78034_o # tabLabel
 public net.minecraft.item.crafting.Ingredient field_193371_b #matchingStacks
-public net.minecraft.world.biome.Biome field_76791_y # biomeName
\ No newline at end of file
+public net.minecraft.world.biome.Biome field_76791_y # biomeName
+public net.minecraft.entity.projectile.EntityFishHook field_190627_av #currentState
```

Comment:
Do we really need them in the JEI AT?
@jaredlll08 I'm not proficient with our multiple AT setup, does he need them in here as well?

Reply:
The ATs need to be on all the projects, if you loaded the JEI project without it, it would crash

Reply:
oop
@expensiveladder could you add the stuff that i put into the AT for the main project into the ATs for the other projects pls?